src/components/StoreItem.js

```diff
@@ -0,0 +1,40 @@
+import { useState } from "react"
+import initialStoreItems from '../store-items'
+import Filters from "./Filters"
+
+const StoreItem = props => {
+    const [storeItems, setStoreItems] = useState(initialStoreItems)
+
+    const addCartItem = item => {
+        for (const inCart of props.cartItems) {
+            if (inCart.id === item.id) {
+                inCart.quantity++
+                return
+            }
+        }
+        const itemsToCart = {
+            id: item.id,
```

Comment:
Could use spread operator here I think?
e.g.
I think spread operator for the props you're not changing might work here?
E.g. {
...itemsToCart,
basePrice: item.price,
totalPrice: item.price,
quantity: 1
}

Reply:
yes totally! thanks!